cypress/integration/admin_institution.js

```diff
@@ -0,0 +1,30 @@
+import { mockUser } from '../../src/utils/testfiles/mock_feide_user';
+
+const authorizedUser = { ...mockUser, 'custom:affiliation': '[member, employee, staff]', email: 'ost@unit.no' }; //@unit.no-address resolves to app admin
+
+describe('User administer institutions ', () => {
+  before('Given that the user is logged in as Application administrator:', () => {
+    cy.visit('/');
+    cy.get('[data-testid=menu-login-button]').click({ force: true });
+    cy.setUserInRedux(authorizedUser);
+  });
+
+  it('The User should be able to open institutions-admin-page from menu', () => {
+    // Open admin-inst-page
+    cy.get('[data-testid=menu]').click({ force: true });
+    cy.get('[data-testid=menu-admin-institution-button]').click({ force: true });
+
+    cy.contains('Institutt for osteloff');
+    cy.contains('Kjetil');
+    cy.contains('27.01.1780');
+  });
+
+  it('The User should be able to add an institution', () => {
+    // Open admin-inst-page
```

Comment:
nødvendig med kommentarer?

Reply:
njaa. tenkte du generelt eller bare dene ? Ser bare at det var gjort på denne måten i de fleste andre spec-filene

Reply:
tenker at når det er så få ting som gjøres i en `it` så er det kanskje ikke nødvendig med kommentarer? men ser det er gjort i de andre testene, så ikke så farlig egentlig. 

men kanskje skriv ut ordene hvis du vil beholde kommentarene? `// Open institution page for admin` osv?

Reply:
Det at det er gjort før er ingen unnskyldning nei ;-)   Men synes det gjør koden mer oversiktlig når koden ikke nødvendigvis er selvforklarende med metodenavn.
Denne test-metoden kommer jo også til å bli lenger. 
Endrer ifallefall til hele ord.